tests/dummy/app/routes/examples/custom-messages.js

```diff
@@ -6,6 +6,7 @@ export default ExampleRoute.extend({
   setupController(controller) {
     this._super(...arguments);
     set(controller, 'themeInstance', BootstrapTheme.create({
+      thead: '',
```

Comment:
Is it needed here?

Reply:
Nope, it is not. I just put it there for testing purposes and accidentally included the file in the commit.